server/models/record.model.ts

```diff
@@ -0,0 +1,45 @@
+import mongoose, { Schema, Document, Model } from 'mongoose';
+import {IChunkScheme} from "./chuck.model";
+import { v4 as uuid } from 'uuid';
+
+export interface IRecord extends Document {
+    recordId: string;
```

Comment:
Because we are already in `record.model` we dont need `recordId`, `id` by itself is enough and its auto-generated by default so we dont need to have a pre ('save') just like the user model